src/statements/IfStatement.py

```diff
@@ -1,10 +1,16 @@
 from grammar.Wuwuzela_GrammarParser import Wuwuzela_GrammarParser
 from src.expressions.LogicalExpression import LogicalExpression
 
+
 class IfStatement():
     def __init__(self, variables, ctx: Wuwuzela_GrammarParser.IfStatementContext):
-        from .Declaration import Declaration    
+        global i
+        from .Declaration import Declaration
         logical_expr = ctx.logicalExpression()
 
         if LogicalExpression(variables, logical_expr).value:
-            Declaration(variables, ctx.declaration())
+            for declaration in ctx.declaration():
+                Declaration(self, declaration)
+
+            if ctx.RBRACE():
```

Comment:
Co się dzieje bez tego?

Reply:
to też można wywalić, na początku miałem błąd który tym naprawiałem ale to było złe podejście